pkg/remote/remotes/docker/fetcher.go

```diff
@@ -151,11 +151,16 @@ func (r dockerFetcher) Fetch(ctx context.Context, desc ocispec.Descriptor) (io.R
 	})
 }
 
-func (r dockerFetcher) createGetReq(ctx context.Context, host RegistryHost, ps ...string) (*request, int64, error) {
+func (r dockerFetcher) createGetReq(ctx context.Context, host RegistryHost, mediatype string, ps ...string) (*request, int64, error) {
```

Comment:
The whole remote package is copied from containerd. Why containerd does not have this problem?

Reply:
It does have this issue, only no client to trigger it:)